src/registrar/tests/test_models_domain.py

```diff
@@ -203,19 +206,22 @@ def test_cache_nested_elements_is_subdomain(self):
     def test_map_epp_contact_to_public_contact(self):
         # Tests that the mapper is working how we expect
         with less_console_noise():
-            domain, _ = Domain.objects.get_or_create(name="registry.gov")
+            domain, _ = Domain.objects.get_or_create(name="registry.gov", state=Domain.State.DNS_NEEDED)
             security = PublicContact.ContactTypeChoices.SECURITY
             mapped = domain.map_epp_contact_to_public_contact(
-                self.mockDataInfoContact,
-                self.mockDataInfoContact.id,
+                self.mockDataSecurityContact,
+                self.mockDataSecurityContact.id,
                 security,
             )
+            print("self.mockDataInfoContact.id is", self.mockDataInfoContact.id)
```

Comment:
you're note using mock data info contact anymore

Reply:
Ah yeah, sorry the code is still in WIP state -  I was testing seeing if creating a new object would help and left the print statement in case the previous code was the right route 😅